### PR TITLE
Added Logistic Regression From Scratch

### PR DESCRIPTION
**Pull request by raypretam**
Resolves issue #8 

### Description
Coded Logistic Regression in Python3 from scratch. Have used Matplotlib to show the loss function and the dataset after classification and used sklearn to generate a random binary classification dataset.

@iam-abbas  please review it.

### CODE REVIEW

**Review by iam-abbas — APPROVED** [review]
This is great!

**Comment by raypretam** [comment]
> This is great!

Thanks a lot. You can open more issues regarding implementing algorithms from scratch. I would love to do the same for other algorithms.

**Comment by iam-abbas** [comment]
You are awesome!
